packages/manager/apps/sign-up/src/form/form.controller.js

```diff
@@ -111,6 +111,8 @@ export default class SignUpFormAppCtrl {
     this.loading.init = true;
 
     const { ovhSubsidiary } = this.$location.search();
+    this.isIndianSubsidiary = ovhSubsidiary;
```

Comment:
I have some doubt `ovhSubsidiary` (`this.$location.search()`) is equivalent to `isIndianSubsidiary`

Reply:
yup, fixed using coreconfig service